### ISSUE TITLE
Update fails when admin account is renamed

### ISSUE THREAD

**Issue by bauxi** [gophish/gophish#1963]
What version of Gophish are you using?: 0.11

Brief description of the issue: When the admin account has been renamed to something else than "admin", the update to 0.11 fails. Gophish won't start. I also mentioned this in <https://github.com/gophish/gophish/issues/1895>.

What are you expecting to see happen? Gophish starting up with my unusual admin account name.

What are you seeing happen? Nothing.

Please provide as many steps as you can to reproduce the problem:
 * Launch Gophish with any version < 0.11 and rename your admin user.
 * Update Gophish to 0.11
 * Try to run Gophish

My workaround for now was to rename my admin user back to "admin" with Gophish 0.10.1 and then perform the upgrade.

**Comment by ironc0in** [comment]
i tried for hours, my user is "root", added an "admin" user, tried running from there, still can't run v0.11 although v0.10 works fine on the same linux machine!

**Comment by GregCFTS** [comment]
Just doing some testing on this, I've noticed that if you don't have an account called "admin" then gophish will refuse to start, with no explanation.

**Comment by bauxi** [comment]
Might be due to the newly introduced password policy and default password generation in 0.11. Maybe Gophish doesn't know what to do if the user for who the password should be generated doesn't exist. This is just a guess though.

**Comment by GregCFTS** [comment]
I agree, I think that is the issue. This gives Jordan an idea of what to test and then how to fix it. If you delete the account and then try to load gophish again it won't work. It must have the admin account in there before it'll work. 

I don't really like having a generic admin account (even with a different password) there, as it gives someone a known account to target.

Would be cool if we could have MFA as an option going forward, might see if anyone has suggested it and if not suggest it myself.

**Comment by h4knet** [comment]
I can confirm I had the same issue.
I would also add to not rename the account event after the upgrade is done correctly as if you have to restart gophish, you will get the same error again...

in other words: if you rename the 'admin' account in v.0.11.0 gophish will not start afterwards.

**Comment by jordan-wright** [comment]
Thanks for the context everyone. I’ve been swamped with other projects but my hope is to get this fixed ASAP.

So sorry for the hassle!
